src/utils/registration-helpers.ts

```diff
@@ -576,12 +576,20 @@ export const contributorConfig: ContributorConfig = {
   },
 };
 
```

Comment:
Var litt i tvil, men landa på at det var like greit å splitte opp denne i to ulike funksjoner nå som det er en case hvor vi bare ønsker en av dem. Kunne ev. latt `groupContributors` leve videre og bare bruke den nye `getContributorsWithPrimaryRole()`, men syns ikke det ble så mye bedre eller

Reply:
Enig i vurderingen din!